convert2rhel/unit_tests/utils_test.py

```diff
@@ -660,6 +660,37 @@ def test_hide_secrets(arguments, secret_args, expected):
     assert sanitazed_cmd == expected
 
 
+def test_hide_secrets_default():
+    """Test that the default secret_args are sound."""
+    test_cmd = [
+        "register",
+        "--force",
+        "--username=jdoe",
+        "--password=Super@Secret@Password",
+        "-p=Super@Secret@Password",
+        "--activationkey=123",
+        "-k=123",
+        "--pool=e6e3f4ca-342f-11ed-b5eb-6c9466263bdf",
+        "--no-rpm-va",
+        "--debug",
+        "--org=0123",
+    ]
+    sanitized_cmd = utils.hide_secrets(test_cmd)
+    assert sanitized_cmd == [
+        "register",
+        "--force",
+        "--username=jdoe",
+        "--password=*****",
+        "-p=*****",
+        "--activationkey=*****",
+        "-k=*****",
+        "--pool=e6e3f4ca-342f-11ed-b5eb-6c9466263bdf",
+        "--no-rpm-va",
+        "--debug",
+        "--org=0123",
+    ]
```

Comment:
I think you could use the `test_hide_secrets`test that has a lot of parameters right now and include this one as another parameter. (I know, sometimes having a test with lots of parameters ins't good, but I think it would be easier for maintaining it later) 

If you want to test that the default sanitization arguments are being sanitized properly, then updating the test to not pass an empty `secret_args` would be more appropriated. 

You don't need to apply this, if you don't it's worth it.